pixart.js

```diff
@@ -1 +1,23 @@
+var brush = document.querySelector( ".brush" );
 
+// I thought this works for both one and two of the commits?
+// After looking around I could only find a jquery solution...
+var setColorClick = function(){
+    button.addEventListener("click" , function(evt){		    button.addEventListener("click" , function(evt){
```

Comment:
I don't understand quite why you are putting the same event listener in the callback function?